src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerPanel.tsx

```diff
@@ -105,6 +108,7 @@ export const TemplatesCatalogDrawerPanel: FC<TemplatesCatalogDrawerPanelProps> =
         .catch((err) => {
           setError(err);
         });
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
no need , recheck please

Reply:
We DO NEED this.

Reply:
please see solution in Dana pr

Reply:
I think Dana's PR is not that solution, it's what you've mentioned regarding Aviv's finding and adding missing useCallback to appropriate places. I haven't seen the PR yet.